tests/integration/test_pr_comment.py

```diff
@@ -1714,7 +1716,9 @@ def test_trigger_packit_command_without_config(
         "For more info, please check out "
         f"[the documentation]({DOCS_HOW_TO_CONFIGURE_URL}) "
         "or [contact the Packit team]"
-        f"({CONTACTS_URL})."
+        f"({CONTACTS_URL}). You can also utilise "
```

Comment:
```suggestion
        f"({CONTACTS_URL}). You can also use "
```

I think *use* sounds more smooth than *utilise* but maybe I am wrong so feel free to let it as it is now :)

🙏🏻